src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ConditionalOrder.java

```diff
@@ -23,4 +36,77 @@
     @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
     private LocalDateTime dateConditionalOrderSubmitted;
```

Comment:
as this will now be prefixed co please rename it dateSubmitted and then write a migration for dateConditionalOrderSubmitted -> coDateSubmitted